verilog/formatting/tree_unwrapper.cc

```diff
@@ -649,7 +649,8 @@ void TreeUnwrapper::Visit(const verible::SyntaxTreeNode& node) {
     case NodeEnum::kAssignmentStatement:             // id=expr
     case NodeEnum::kProceduralTimingControlStatement: {
       if (IsTopLevelListItem(Context())) {
-        VisitNewUnwrappedLine(node);
+        VisitIndentedSection(node, 0,
+                             PartitionPolicyEnum::kFitOnLineElseExpand);
```

Comment:
This change should be pretty benign, unless there are some cases not covered by unit tests.
I didn't update the `tree_unwrapper` tests for now so they obviously fail, but all other (including end-to-end formatting) pass.

Reply:
This makes sense.  In some cases of statements in tree_unwrapper_test.cc, you can probably update by replacing `L(k, ...)` with `NL(k, ...)` which pushes a leaf partition down one level.  I don't have any `formatter_test` test cases yet that uses `// verilog-syntax: parse-as-statements`, which invokes:

https://github.com/google/verible/blob/78170927a2f881a6189a71a871b62701eb6ddfa0/verilog/analysis/verilog_excerpt_parse.cc#L72
https://github.com/google/verible/blob/78170927a2f881a6189a71a871b62701eb6ddfa0/verilog/analysis/verilog_excerpt_parse.cc#L106

but you're welcome to try it out.  That comment allows the parser to parse a source file containing only statements.